app/styles/front.scss

```diff
@@ -306,17 +306,15 @@ $autre-lighter: #cce4e7;
   border-left-width: 4px;
   margin-bottom: 10px;
 
-  h2 {
```

Comment:
Changing instructions for such a generic selector is extremely dangerous. Have you visited _every instance of `<h2>` throughout the app_ after having made the change?